SOURCE/Main_DOODZ.c

```diff
@@ -806,6 +756,11 @@ int RunMDOODZ( int nargs, char *args[] ) {
                 if ( model.Newton == 1 && Nmodel.nit > 0 ) RheologicalOperators( &mesh, &model, &scaling, 1 );
                 if ( IsJacobianUsed == 1 )                  BuildJacobianOperatorDecoupled( &mesh, model, 0, mesh.p_corr, mesh.p_in, mesh.u_in, mesh.v_in,  &Jacob,  &JacobA,  &JacobB,  &JacobC,   &JacobD, 1 );
                 
+                // IsNanArray2DFP(mesh.eta_n, (mesh.Nx-1)*(mesh.Nz-1));
```

Comment:
Why do we comment those lines if we can just delete them?

Reply:
I needed to activate it yesterday for debugging. Could be removed also.